addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -293,11 +298,30 @@ var SlideUploadDialog = Dialog.extend({
             fill_data: function (query, data) {
                 var that = this,
```

Comment:
and just for the lol : 
why not `var those = [this, that, self]`

(don't worry, I know it's not you :D)

Reply:
Do you want me to change it or should I keep the funny stuff?

Reply:
while you are at it, yes, as it's master branch, you can :)